WordPress/src/main/res/values/strings.xml

```diff
@@ -1594,6 +1594,8 @@
     <string name="error_media_canceled">Uploading media were canceled</string>
     <string name="error_media_quota_exceeded">Storage quota exceeded</string>
     <string name="error_media_quota_exceeded_toast">Cannot upload file.\nStorage quota was exceeded.</string>
+    <string name="error_media_xmlrpc_not_allowed">Operation not allowed</string>
```

Comment:
Could we maybe make this a bit more explicit? Maybe something like "Operation not allowed by XMLRPC"?